clang builds fail to compile with OpenMP

**Describe the bug**
Trying to use `clang` with `OpenMP` results in:
```
-- Using OpenMP for worker threading
-- Could NOT find OpenMP_C (missing: OpenMP_C_FLAGS OpenMP_C_LIB_NAMES) 
-- Could NOT find OpenMP_CXX (missing: OpenMP_CXX_FLAGS OpenMP_CXX_LIB_NAMES) 
-- Could NOT find OpenMP (missing: OpenMP_C_FOUND OpenMP_CXX_FOUND) 
-- OpenMP has been found: Linker="", CC FLAGS="NOTFOUND", CXX FLAGS="NOTFOUND"
CMake Error at cmake/load_threading_package.cmake:26 (message):
  requested OpenMP with -DUSE_OPENMP=On, but cannot find valid OpenMP in
  compiler
Call Stack (most recent call first):
  cmake/load_packages.cmake:13 (include)
  CMakeLists.txt:61 (include)
```

This could possibly be fixed with installing `libomp5-dev` in clang container.

**To Reproduce**
Use `VT_USE_OPENMP` / `USE_OPENMP` configuration variables.

**Expected behavior**
Make sure that CI passes with one of `clang` builds using `OpenMP`.

See #1214 for recent changes in the area.

--- a/ci/docker/ubuntu-18.04-clang-cpp.dockerfile
+++ b/ci/docker/ubuntu-18.04-clang-cpp.dockerfile
@@ -24,6 +24,7 @@ RUN apt-get update -y -q && \
     valgrind \
     make-guile \
     libomp5 \
+    libomp-dev \
     ccache && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*

--- a/ci/docker/ubuntu-20.04-clang-cpp.dockerfile
+++ b/ci/docker/ubuntu-20.04-clang-cpp.dockerfile
@@ -24,6 +24,7 @@ RUN apt-get update -y -q && \
     valgrind \
     make-guile \
     libomp5 \
+    libomp-dev \
     ccache && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*

